concepticondata/conceptlists.tsv

```diff
@@ -441,4 +441,5 @@ Smith-2017-679	Smith, Alexander D.	2017		679	basic	English	Burneo languages		Smi
 Zalizniak-2024-4583	Anna Zalizniak and Anna Smirnitskaya and Maksim Russo (Rousseau) and Ilya Gruntov and Timur Maisak and Dmitry Ganenkov and Maria Bulakh and Maria Orlova and Marina Bobrik-Fremke and Oksana Dereza and Tatiana Mikhailova and Maria Bibaeva and Mikhail Voronov	2024		4583	ranked	English	global	http://datsemshift.ru	Zalizniak2024		This is the most recent dump of the DatSemShift database, which was retrieved on February 5 (2024) from the database and then converted to CLDF. The data contains several duplicates, which we have marked by a preceding asterisk, leaving them unlinked to Concepticon and also ignoring them in the network representations.		DatSemShift
 Lastra-1986-381	Lastra de Suárez, Yolanda	1986		381		English	Spanish		Lastra1986		This concept list is the result of a study of the dialectal regions in modern Nahuatl. 	239-248	
 Tjuka-2024-1310	Tjuka, Annika	2024		1310		English	Vietnamese		Tjuka2024		The list includes 1310 concepts for Vietnamese. The concepts are based on the items provided in the [Intercontinental Dictionary Series](:ref:Key-2016-1310).		
-Snodgrass-1980-260	Snodgrass, Joan G. and Vanderwart, Mary	1980		260	basic, naming test	English	global 	https://psycnet.apa.org/doi/10.1037/0278-7393.6.2.174	Snodgrass1980		This list comprises 260 concepts from several widely studied semantic categories that are to be elicited using a standardized set of 260 pictures for use in experiments investigating differences and similarities in the processing of pictures and words. The pictures are black-and-white line drawings executed according to a set of rules that provide consistency of pictorial representation.		
+Snodgrass-1980-260	Snodgrass, Joan G. and Vanderwart, Mary	1980		260	basic, naming test	English	global 	https://psycnet.apa.org/doi/10.1037/0278-7393.6.2.174	Snodgrass1980		This list comprises 260 concepts from several widely studied semantic categories that are to be elicited using a standardized set of 260 pictures for use in experiments investigating differences and similarities in the processing of pictures and words. The pictures are black-and-white line drawings executed according to a set of rules that provide consistency of pictorial representation.
+MorenoMartinez-2012-360	Moreno-Martinez, Francisco J. and Montoro, Pedro R.	2012		360	naming test	English, Spanish	global	https://journals.plos.org/plosone/article/file?id=10.1371/journal.pone.0037527&type=printable	MorenoMartinez2012		This work presents a new set of 360 high quality color images belonging to 23 semantic subcategories. These were named by 230 Spanish native speakers and come with multiple psycholinguistic variables, here: AoA, familiarity, typicality, visual complexity and lexical frequency.		
```

Comment:
The URL points to the PDF of the article, but the URL is usually used to link the original data. Do you happen to have the data as well, a link to it? So I'd suggest to replace the link.

Reply:
swapped for a link that points to the data and norms :)